sdk/typescript/src/index.ts

```diff
@@ -236,6 +258,8 @@ export {
 	Inputs,
 	/** @deprecated Import from `@mysten/sui.js/transactions` instead */
 	TransactionBlock,
+	/** @deprecated Import from `@mysten/sui.js/transactions` instead */
```

Comment:
Honestly this is exported by mistake, we really shouldn't even export this

Reply:
I'd be fine just marking as `deprecated` with no message because there's no real way to replace it

Reply:
this is used in a lot of places, but I don't really understand the transaction building stuff well enough to know why we can't replace it

Reply:
Outside of the SDK or inside of it?

Reply:
it's used once in the wallet, and a bunch in the kiosk sdk

Reply:
Hmm okay we can revisit later. I might be wrong and this might be useful

Reply:
WE probably just eventually want to make this a type-only export

Reply:
already made the new export type-only